### ISSUE TITLE
Docs should elevate information on adding footprints

### ISSUE THREAD

**Issue by jbullion-astra** [atopile/atopile#430]
Currently the info is under the [install dependencies](https://atopile.io/install/#adding-custom-footprints). This information should either be promoted to its own page, or at least have a link in the how to section.

**Comment by mawildoer** [comment]
Thanks for the suggestion! You're right - this is all a little obfuscated. Does this explain what you're looking for? https://github.com/atopile/atopile/pull/431

**Comment by jjrreett** [comment]
It’s certainly better than it was! Might want to link or update the install page. 

I have been dropping my 'kicad_mod' files into the build directory. My take on your docs is that any mod file in src will get picked up?

This would also be a good place to note how pin naming correlates with the footprints.

**Comment by mawildoer** [comment]
Yeah, great point. Made those updates as well. Thanks for raising it! 🙌
